ecosystem/sep-direct-payment.md

```diff
@@ -0,0 +1,318 @@
+1. The sending client chooses a destination region.
+1. The sending anchor fetches the TOML file to find the SEND_SERVER API and [`/info`](#info) endpoint fields for the receiving anchor for this region.
+1. Using the fields fetched from /info, the sending anchor collects all needed information from the sending client.
+1. The sending anchor POSTs all that information to the [`/send](#send) endpoint of the receiving anchor.
+1. If information is missing, or the receiving anchor needs extra information to complete this transaction, `POST /send` will return `403 Customer Information Needed` along with the missing fields. The sending anchor should collect these and retry the entire `POST /send` call, until it succeeds.
+1. Once all needed information is provided and the receiving anchor is ready to complete this transaction, the `POST /send` call will return a `200` status along with information needed to complete the transaction over the stellar network.
+1. The sending anchor should collect the fiat from the sending client, and perform the specified stellar transaction to send the money to the receiving anchor.  This is usually a path payment, but can be done with a regular payment as well, so long as the receiving anchor gets the token they expect.
+1. Once the stellar transaction is completed, the receiving anchor should deposit the money in the receivers bank account.
+1. The sending anchor can query the status of this transaction via the [`/transaction`](#transaction) endpoint, and should communicate updates to the sending client as that progresses.
+
+
+
+## API Endpoints
+
+* [`GET /info`](#info): required
+* [`POST /send`](#send): required
+* [`GET /transactions`](#transaction-history): required
+* [`GET /transaction`](#single-historical-transaction): required
+
+### Info
+#### Request
+
+```
+GET DIRECT_PAYMENT_SERVER/info
+```
+
+Allows an anchor to communicate basic info about what currencies their `DIRECT_PAYMENT_SERVER` supports receiving from partner anchors.
+
+Request parameters:
+
+Name | Type | Description
+-----|------|------------
+`lang` | string | (optional) Defaults to `en`. Language code specified using [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1). `description` fields in the response should be in this language.
+
+#### Response
+
+The response should be a JSON object like:
+
+```json
+{
+   "receive":{
+      "USD":{
+         "enabled":true,
+         "fee_fixed":5,
+         "fee_percent":1,
+         "min_amount":0.1,
+         "max_amount":1000,
+         "fields":{
+            "sender":{
+               "first_name":"The sender's first name",
+               "last_name":"The sender's last name"
+            },
+            "receiver":{
+               "first_name":"The receiver's first name",
+               "last_name":"The receiver's last name",
+               "email_address":"The receiver's email address"
+            },
+            "transaction":{
+               "routing_number":{
```

Comment:
Are the fields here from a SEP? I don't see it in SEP-9, but I see `bank_number`. Ideally anchors would only use fields from a SEP, otherwise Anchor A could list `routing_number` but Anchor B could list `routing_num` and the sending Anchor C would have to know how to handle both, which would get really messy quickly.

Reply:
There's no SEP for transfer information but we can either create one, or add such fields in this SEP